pkg/util/version/version.go

```diff
@@ -4,7 +4,7 @@ package version
 // Licensed under the Apache License 2.0.
 
 const (
-	OpenShiftVersion    = "4.3.10"
-	OpenShiftPullSpec   = "quay.io/openshift-release-dev/ocp-release@sha256:edb4364367cff4f751ffdc032bc830a469548f998127b523047a8dd518c472cd"
-	OpenShiftMustGather = "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:7b4b7938d33cc9f86d822cf6a4dbffb99444566ef93f45a313f600cad11cfee3"
+	OpenShiftVersion    = "4.3.12"
+	OpenShiftPullSpec   = "quay.io/openshift-release-dev/ocp-release@sha256:75e8f20e9d5a8fcf5bba4b8f7d17057463e222e350bcfc3cf7ea2c47f7d8ba5d"
+	OpenShiftMustGather = "quay.io/openshift-release-dev/ocp-v4.0-art-dev@sha256:a37e5bef81b80c86ab3864c3395d69c0867ab0aa58e1150eceb85be8951def49"
```

Comment:
where did you get that ID from? shouldn't it be `dec10f73956f9db4e4e6c878c1dcacba0f17ebe4146c2032d2005f57ad120d0c`?

Reply:
I think I misscopied from `oc adm release info`. we might need to automate it somehow